xpcPlugin/MessageHandlers.cpp

```diff
@@ -573,7 +573,7 @@ namespace XPC
 
 		if (aircraftNumber > 0)
 		{
-			// Enable AI for the aircraftNumber we are setting
+			// Enable AI for the aircraftNumber we are setting: MRT COMMENT: ENABLE, OR DISABLE?
```

Comment:
Ah, yes, I see.  I was looking for the AI disable within the code itself, which is why I was confused.  This is a case that I'm actually looking to explore with my code.  Not sure the plugin needs updating though, I just need to disable the AI after moving the a/c.

That could be something helpful to mention in the Wiki though.  That the sendPOSI command automatically turns the AI autopilot back on.